bus-mapping/src/evm/opcodes/callop.rs

```diff
@@ -173,9 +176,8 @@ impl<const N_ARGS: usize> Opcode for CallOpcode<N_ARGS> {
         let is_precompile = code_address
             .map(|ref addr| is_precompiled(addr))
             .unwrap_or(false);
-        // TODO: What about transfer for CALLCODE?
         // Transfer value only for CALL opcode, is_precheck_ok = true.
-        if call.kind == CallKind::Call && is_precheck_ok {
+        if call.kind == CallKind::Call && is_precheck_ok && !call.value.is_zero() {
```

Comment:
Suggestion to update the comment above with the new condition.

Reply:
Nice catch. Fixed here 6395ad6